xml/System.Security/PartialTrustVisibilityLevel.xml

```diff
@@ -33,7 +33,7 @@
   
  You enable partially trusted assemblies that are identified as NotVisibleByDefault by adding them to the <xref:System.AppDomainSetup.PartialTrustVisibleAssemblies%2A> property of their application domain. If you enable an assembly that references (directly or indirectly) other partially trusted assemblies that are NotVisibleByDefault, those other assemblies should be enabled as well.  
   
- When an APTCA library that specifies a <xref:System.Security.PartialTrustVisibilityLevel.NotVisibleByDefault> and that is eligible for code sharing is loaded for the first time, it is loaded into the shared domain. Whenever that assembly is loaded with the same <xref:System.Security.PartialTrustVisibilityLevel> into another domain, it will be shared. However, if the assembly is loaded with a different <xref:System.Security.PartialTrustVisibilityLevel.NotVisibleByDefault>, it will not be shared.  
+ When an APTCA library that specifies a <xref:System.Security.PartialTrustVisibilityLevel> and that is eligible for code sharing is loaded for the first time, it is loaded into the shared domain. Whenever that assembly is loaded with the same <xref:System.Security.PartialTrustVisibilityLevel> into another domain, it will be shared. However, if the assembly is loaded with a different <xref:System.Security.PartialTrustVisibilityLevel>, it will not be shared.  
```

Comment:
```suggestion
 When an APTCA library that specifies a `PartialTrustVisibilityLevel` and that is eligible for code sharing is loaded for the first time, it is loaded into the shared domain. Whenever that assembly is loaded with the same `PartialTrustVisibilityLevel` into another domain, it will be shared. However, if the assembly is loaded with a different `PartialTrustVisibilityLevel`, it will not be shared.  
```